pkg/apis/openebs.io/lvm/v1alpha1/lvmnode.go

```diff
@@ -67,6 +67,45 @@ type VolumeGroup struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Minimum=0
 	PVCount int32 `json:"pvCount"`
+
+	// MaxLV denotes maximum number of Logical volumes allowed
+	// in volume group or 0 if unlimited.
+	MaxLV int32 `json:"maxLv"`
+
+	// MaxPV denotes Maximum number of physical volumes allowed
+	// in volume group or 0 if unlimited.
+	MaxPV int32 `json:"maxPv"`
+
+	// SnapCount denotes number of snapshots in volume group.
+	SnapCount int32 `json:"snapCount"`
+
+	// MissingPVCount denotes number of physical volumes in
+	// volume group which are missing.
+	MissingPVCount int32 `json:"missingPvCount"`
+
+	// MetadataCount denotes number of metadata areas on the
+	// volume group.
+	MetadataCount int32 `json:"metadataCount"`
+
+	// MetadataUsedCount denotes number of metadata areas in
+	// volume group
+	MetadataUsedCount int32 `json:"metadataUsedCount"`
+
+	// MetadataFree specifies the available metadata area space
+	// for the volume group
+	MetadataFree resource.Quantity `json:"metadataFree"`
+
+	// MetadataSize specifies size of smallest metadata area
+	// for the volume group
+	MetadataSize resource.Quantity `json:"metadataSize"`
+
+	// Permission indicates the volume group permission
+	// which can be writable or read-only
+	Permission int `json:"permissions"`
+
+	// AllocationPolicy indicates the volume group allocation
+	// policy(normal/contiguous/cling/anywhere/inherited)
+	AllocationPolicy int `json:"allocationPolicy"`
```

Comment:
Why do we need the above info in lvmnode? How is the data refreshed?

Reply:
All these correspond to VolumeGroup struct which is being used by the LVMNode CRD.

Reply:
About refreshing not sure how it is been taken care of.

Reply:
There is a controller for this resource which updates after fetching using `vgs` command

Reply:
For allocation policy, can you give a description of which integer corresponds to which policy. Also, could think of using a custom type and declaring the constants as `iota`

Reply:
How would we be able to compare the values that we get from vgs and are custom defined enums?